packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -790,12 +790,15 @@ export class UnfurlService extends BaseService {
 
             const urlWithoutParams = url.split('?')[0];
             const exploreModel = urlWithoutParams.split('/tables/')[1];
-
+            const internalUrl = url.replace(
```

Comment:
converts external URL to internal URL by replacing siteUrl with internal host. Regular case scenario, both variables are the same, and nothing changes.